plugins/vpp/ifplugin/interface_config.go

```diff
@@ -252,6 +253,23 @@ func (plugin *InterfaceConfigurator) ConfigureVPPInterface(iface *intf.Interface
 		errs = append(errs, err)
 	}
 
+	// rx placement todo simplify implementation when the binary api call will be available (remove dump)
```

Comment:
Please write TODOs in following format: `// TODO: ...`

This allows automatic tools to catch them and also readers to more easily see them.